src/components/views/right_panel/UserInfo.tsx

```diff
@@ -1447,18 +1468,26 @@ const BasicUserInfo: React.FC<{
     const userTrust = useAsyncMemo<UserVerificationStatus | undefined>(
         async () => cli.getCrypto()?.getUserVerificationStatus(member.userId),
         [member.userId],
+        // the user verification status is not initialized
+        undefined,
```

Comment:
isn't this the default? Not that it matters to make it explicit, I guess

Reply:
Yes this default but I wanted to make it explicit here